code/go/0chain.net/smartcontract/zcnsc/handler.go

```diff
@@ -72,11 +71,6 @@ func (zrh *ZcnRestHandler) getAuthorizerNodes(w http.ResponseWriter, r *http.Req
 		return
 	}
 
-	if len(authorizers) == 0 {
```

Comment:
why do you want to remove this check?

Reply:
It's more natural to receive empty array instead of error, if no authorizers are registered. The logic, which handles not enough number of authorizers has already been implemented in gosdk: https://github.com/0chain/gosdk/blob/f0d08895c0f1a0abd22f884a8d2fa767c2acbf25/zcnbridge/authorizers_query.go#L53-L55